packages/jazz-tools/src/tools/auth/clerk/index.ts

```diff
@@ -77,6 +78,7 @@ export class JazzClerkAuth {
     if (!clerkClient.user) {
       if (isAuthenticated) {
         this.authSecretStorage.clear();
+        await this.logOut();
```

Comment:
If we are calling logOut we don't need to call `authSecretStorage.clear`

```suggestion
        await this.logOut();
```